src/components/notes/AddNote.js

```diff
@@ -0,0 +1,116 @@
+import { TextField, Card, CardActions, Button, useTheme, Divider, Typography } from '@material-ui/core'
+import {MarkdownEditor} from '../commons/MarkdownEditor'
+import {useState} from 'react'
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
+import { TagsList } from './TagsList'
+import '../../App.css';
+import { useDevMail } from '../../utils/hooks/useDevMail';
+import { encode } from '../../utils/hooks/Base64';
+import { useDevPassword } from '../../utils/hooks/useDevPassword';
+import { NoteApi } from '../../utils/apis/NoteApi';
+import Swal from 'sweetalert2'
+import {withSwal} from '../commons/formsCommons/WithSwal'
+
+export const AddNote = ({ onCancel, uuid, uuid_key, reload, setReload }) => {
+
+    const theme = useTheme()
+    const {getEncoded} = useDevPassword()
```

Comment:
W sumie to tak się zastanawiam czy takie rzeczy jak wyciąganie z `localStorage` powinny być hookami, ale to taka dygresja poza reviewowa

Reply:
Znaczy ogólnie nie widzę problemu, dlaczego nie. useCośtam to tylko nazwa funkcji